<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DecisionMakerWidget</class>
 <widget class="QWidget" name="DecisionMakerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="operationsTab">
      <attribute name="title">
       <string>Operations</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="chartTab">
      <attribute name="title">
       <string>Chart</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="logsTab">
      <attribute name="title">
       <string>Logs</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="portfolioTableWidget">
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Available</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Avg price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cost</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Part</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Income</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Income, %</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>In a day, %</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
